organization/src/main/scala/com/improving/app/organization/domain/OrganizationValidation.scala

```diff
@@ -0,0 +1,24 @@
+package com.improving.app.organization.domain
+
+import com.improving.app.common.errors.Validation._
+import com.improving.app.common.errors.ValidationError
+
+object OrganizationValidation {
+  val organizationNameValidator: Validator[String] = name => {
+    if (name.isEmpty) {
+      Some(ValidationError("name empty"))
+    } else {
+      None
+    }
+  }
+
+  val completeOrganizationInfoValidator: Validator[OrganizationInfo] =
+    applyAllValidators[OrganizationInfo](Seq(
+      orgInfo => organizationNameValidator(orgInfo.name),
+      orgInfo => organizationNameValidator(orgInfo.shortName),
+      orgInfo => required("tenant", tenantIdValidator)(orgInfo.tenant),
```

Comment:
Here is where None.get can occur

Reply:
How? 

> sample-requests.txt for the organization service?

We need a better way of handling that, a directory with one request per file or something.

Reply:
One example where this exception can occur is when running a grpcurl command in the command line, but without the tenantId field. The logs will show None.get. I'm not sure if it happens exactly in this line, but it happens somewhere 🤔

Reply:
Found that in the ServiceImpl . We will probably want to check other services for validation at the service level too.